DiscordBot.Modules/CommandModules/PollModule.cs

```diff
@@ -110,5 +118,19 @@ public async Task EndPollAsync(string idOrUrl, [Remainder] string message = "")
                 //await err.DeleteAsync();
             }
         }
+
+        private bool CheckPollMsg(IUserMessage msg)
+        {
+            //Console.WriteLine($"`{msg.Embeds.First().Fields.Count()}Desc");
+            if ((msg.Content != "") ||
+                (msg.Embeds.Count != 1) ||
+                (msg.Embeds.First().Title == "") ||
+                (msg.Embeds.First().Footer.ToString() != $"{_prefix}poll") ||
+                (msg.Embeds.First().Timestamp == null) ||
+                (msg.Embeds.First().Description != null) ||
+                (msg.Embeds.First().Fields.Count() != 1) ||
+                (msg.Embeds.First().Fields.First().Value == "")) { return false; }
+            return true;
+        }
```

Comment:
Das ganze in mehrere checks(abfragen) aufteilen und mit return false; arbeiten.

Reply:
Wie meinst du, mehrere checks?
Mehrere ifs?